deployments/engine/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "inference-manager-dispatcher.serviceAccountName" . }}
```

Comment:
"inference-manager-engine.serviceAccountName"?

Reply:
ah, sorry, forgot to cc. :( https://github.com/llm-operator/inference-manager/pull/173/files fixed.